packages/design-system/src/components/web-components/preactement/define.ts

```diff
@@ -150,6 +191,8 @@ function onConnected(this: CustomElement) {
   const json = this.querySelector('[type="application/json"]');
   const data = parseJson.call(this, props || json?.innerHTML || '{}');
 
+  const eventCallbacks = proxyEvents(this.__properties, this.__options.events, this);
```

Comment:
I also think we could name this a little better to make it clear that these are the names of the event-handler props. Maybe even just calling it `eventHandlers` would help, because that's what they're called in React.